.github/workflows/make-release.yml

```diff
@@ -0,0 +1,48 @@
+name:  Image release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build_and_push_to_ghcr:
+    name: Create docker release and publish to ghcr
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.20']
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: ${{ matrix.go }}
+    - name: checkout
+      uses: actions/checkout@v4
+    - name: Run docker build
+      run: make docker-build
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Build and push cl-dataplane image
+      run: |
```

Comment:
does it make sense to have a make target for that? Similar to L23 above which runs `make docker-build`.
nit: also, the name on the above line (L30) should probably not include `build` in it as it only tags and pushes.